2020/day10/solution.py

```diff
@@ -0,0 +1,28 @@
+from collections import Counter
+from typing import Iterable
+
+def compute_gaps(numbers : Iterable[int]) -> Counter:
+    current = 0
+    gaps = Counter()
+    for num in numbers:
+        gaps[num - current] += 1
+        current = num
+    # add one more gap 3 for the final jump
+    gaps[3] += 1
+    return gaps
+
+def count_arrangements(numbers : Iterable[int]) -> int:
+    target = max(numbers)
+    counter = Counter()
+    counter[0] = 1
+    for i in range(1, target+1):
+        if i in numbers:
+            counter[i] = counter[i - 1] + counter[i - 2] + counter[i-3]
+    return counter[target]
+
+with open('input.txt') as f:
+    numbers = sorted(int(line.strip()) for line in f)
+gap_counter = compute_gaps(numbers)
```

Comment:
```suggestion
    numbers = {int(line.strip()) for line in f}
gap_counter = compute_gaps(numbers)
```
In `compute_gaps` you can do `for num in sorted(numbers)` - I was really confused at first until I saw the sorted line here in the input parsing.